client/src/containers/App.js

```diff
@@ -17,6 +18,7 @@ class App extends Component {
 
     logout() {
         this.props.dispatch(logoutAndRedirect());
+        this.props.dispatch(clearCode());
```

Comment:
Not really important, but might be good to extract dispatch from this.props instead of having this function chain.